File: JAVA/Kode Program/Aksesoris.java
public class Aksesoris extends PetShop {
    private String jenis;   // Jenis aksesoris (kalung, mainan, dll.)
    private String bahan;   // Bahan aksesoris (plastik, kain, logam, dll.)
    private String warna;   // Warna aksesoris

    // Konstruktor default
    public Aksesoris() {
        super();  // Memanggil konstruktor default dari PetShop
        this.jenis = "";
        this.bahan = "";
        this.warna = "";
    }

    // Konstruktor dengan parameter
    public Aksesoris(int id, String nama_produk, double harga_produk, int stok_produk, String jenis, String bahan, String warna) {
        super(id, nama_produk, harga_produk, stok_produk); // Memanggil konstruktor dari PetShop
        this.jenis = jenis;
        this.bahan = bahan;
        this.warna = warna;
    }

    // Setter methods
    public void set_jenis(String jenis) {
        this.jenis = jenis;
    }

    public void set_bahan(String bahan) {
        this.bahan = bahan;
    }

    public void set_warna(String warna) {
        this.warna = warna;
    }

    // Getter methods
    public String get_jenis() {
        return this.jenis;
    }

    public String get_bahan() {
        return this.bahan;
    }

    public String get_warna() {
        return this.warna;
    }

}

File: PYTHON/Kode Program/Aksesoris.py
from PetShop import PetShop

class Aksesoris(PetShop):
    __jenis = ""
    __bahan = ""
    __warna = ""

    def __init__(self, id=0, nama_produk="", harga_produk=0, stok_produk=0, jenis="", bahan="", warna=""):
        super().__init__(id, nama_produk, harga_produk, stok_produk)
        self.__jenis = jenis
        self.__bahan = bahan
        self.__warna = warna

    # Setter methods
    def set_jenis(self, jenis):
        self.__jenis = jenis

    def get_jenis(self):
        return self.__jenis

    def set_bahan(self, bahan):
        self.__bahan = bahan

    def get_bahan(self):
        return self.__bahan

    def set_warna(self, warna):
        self.__warna = warna

    def get_warna(self):
        return self.__warna
